dts/bindings/disk/zephyr,ram-disk.yaml

```diff
@@ -31,5 +31,16 @@ properties:
     type: phandle
     description: |
       Optional phandle to the memory region to be used as a RAM disk,
-      if not used a local buffer is defined for each disk instance.
-      Use it with caution as it makes memory contents easily accessible.
+      if not used, and 'disk-image' property is also not used, a local buffer is
+      defined for each disk instance. Use it with caution as it makes memory 
+      contents easily accessible.
+
+  disk-image:
+    type: string
+    description: |
+      Optional path of a disk image, on the build host system, to be used as
+      backing store of a RAM disk. Make sure that 'sector-size' and 'sector-count'
+      are set to correct values such that 'sector-size' matches the corresponding
+      value used in the existing filesystem (if any) on the disk image and the
+      computed disk size (sector-size x sector-count) matches the size of the disk
+      image in bytes.
```

Comment:
```suggestion
      image in bytes.

```

Reply:
IIRC, I had specifically made sure to add a blank line here but turns out it is not there after commit. But after you suggestion here, I just tried adding it again and force pushed my PR branch - same result. Looks like Git is stripping the last empty line here. I have checked several other .yaml files under bindings subtree and all have no empty line at the end either. So I guess I'll leave it as it is, but if it is a must have, let me know (I am new to Zephyr devel community).

Reply:
@kartben Yes, I suggest adding an newline, even if it is not displayed.

Reply:
So you want two blank lines? Good luck with getting the compliance check to be happy with that (as per the latest failure in CI)...
Please re-read my comment again carefully.

@hIAhmad apologies for the noise, you had it right the first time around (and thanks for this!)

Reply:
No, I want a line, but I'm not sure whether the author added it as required, so I marked it on purpose. But the author added it, and it did a good job.

Reply:
Okay, I'll remember it. It's very puzzling :)